packages/grafana-data/src/themes/createVisualizationColors.ts

```diff
@@ -46,7 +46,7 @@ export function createVisualizationColors(colors: ThemeColors): ThemeVisualizati
 
   const byNameIndex: Record<string, string> = {};
 
-  for (const hue of hues) {
+  hues.forEach((hue) => {
```

Comment:
For some unknown reason, the peer deps change make storybook build fail due to some exception being raise trying to transform this for, replacing with a foreach make storybook build properly again 🤷 

Related: https://github.com/babel/babel/issues/9530